ui/src/config/section/compute.js

```diff
@@ -299,7 +299,7 @@ export default {
           docHelp: 'adminguide/virtual_machines.html#how-to-dynamically-scale-cpu-and-ram',
           dataView: true,
           show: (record) => { return ['Stopped'].includes(record.state) || (['Running'].includes(record.state) && record.hypervisor !== 'KVM' && record.hypervisor !== 'LXC') },
-          disabled: (record) => { return !record.isdynamicallyscalable },
+          disabled: (record) => { return !record.isdynamicallyscalable && record.state !== 'Stopped' },
```

Comment:
Why not `disabled: (record) => { return record.state === 'Running' && !record.isdynamicallyscalable }`

Reply:
> Maybe
> `disabled: (record) => { return record.state === 'Running' && !record.isdynamicallyscalable }`

but isnt `record.state !== 'Stopped'` and `record.state === 'Running' ` are same?

Reply:
@ravening In this case yes. My point was to check VM's state first and isdynamicallyscalable later as that is relevant only for running VMs.

Reply:
Anyway, I'm fine with original change as well

Reply:
updated. thanks @shwstppr @ravening